README.md

```diff
@@ -206,19 +198,11 @@ Example:
 
```

Comment:
**[Optional]**

- [ ] Kindly add **Key Features** and **Future Feature** section are still empty :basket: Kindly add fill these sections to make your readme descriptive :+1: